understanding/captions-live.html

```diff
@@ -58,198 +56,143 @@
                <div class="box-i">
                   <ul>
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.4:</span>Captions (Live) (Level AA)
+
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.4:</span>キャプション (ライブ) (レベル AA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
                   <p>
-                     <a href="#dfn-captions">Captions</a> are provided for all <a href="#dfn-live">live</a>
-                     <a href="#dfn-audio">audio</a> content in <a href="#dfn-synchronized-media">synchronized media</a>.
+                     <a href="#dfn-synchronized-media">同期したメディア</a>に含まれているすべての<a href="#dfn-live">ライブ</a>の<a href="#dfn-audio">音声</a>コンテンツに対して<a href="#dfn-captions">キャプション</a>が提供されている。
+
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Live videos have captions.</dd>
+                  <dt>目標</dt>
+                  <dd>ライブの映像にキャプションがあるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide synchronized text for audio content in real-time videos.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>リアルタイムの映像の音声コンテンツに同期したテキストを提供する。</dd>
```

Comment:
「映像の音声」という字面がどうにも読みにくく感じます……。たとえば in を「含まれる」くらいにしても良いのかもしれません。

```suggestion
                  <dd>リアルタイムの映像に含まれる音声コンテンツに、同期したテキストを提供する。</dd>
```

Reply:
@bakera ああ、たしかに、in を「に含まれる」と訳したほうが、ここではよさそうですね。